tests/devlxd-vm

```diff
@@ -20,7 +20,12 @@ echo "==> Create storage pool using driver ${poolDriver}"
 lxc storage create "${poolName}" "${poolDriver}"
 
 echo "==> Create VM and boot"
-lxc launch "${IMAGE}" v1 --vm -s "${poolName}"
+if hasNeededAPIExtension devlxd_images; then
```

Comment:
`devlxd_images` is in `4.0/stable`:

```
root@f1:~# lxc --version
4.0.10
root@f1:~# lxc info | grep devlxd_images
- devlxd_images
```

To be amended with `devlxd_images_vm` when it's a thing?

Reply:
Yep, fixed :)